src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6144,52 +6104,54 @@ void func_8083F070(Player* this, LinkAnimationHeader* anim, GlobalContext* globa
 
 s32 func_8083F0C8(Player* this, GlobalContext* globalCtx, u32 arg2) {
     CollisionPoly* wallPoly;
-    Vec3f sp50[3];
-    f32 sp4C;
-    f32 phi_f2;
-    f32 sp44;
-    f32 phi_f12;
+    Vec3f wallVertices[3];
+    f32 tempX;
+    f32 temp;
+    f32 tempZ;
+    f32 maxWallZ;
     s32 i;
 
     if (!LINK_IS_ADULT && !(this->stateFlags1 & PLAYER_STATE1_27) && (arg2 & 0x30)) {
         wallPoly = this->actor.wallPoly;
-        CollisionPoly_GetVerticesByBgId(wallPoly, this->actor.wallBgId, &globalCtx->colCtx, sp50);
+        CollisionPoly_GetVerticesByBgId(wallPoly, this->actor.wallBgId, &globalCtx->colCtx, wallVertices);
 
-        sp4C = phi_f2 = sp50[0].x;
-        sp44 = phi_f12 = sp50[0].z;
+        // compute min and max x/z of wall vertices
+        tempX = temp = wallVertices[0].x;
+        tempZ = maxWallZ = wallVertices[0].z;
         for (i = 1; i < 3; i++) {
-            if (sp4C > sp50[i].x) {
-                sp4C = sp50[i].x;
-            } else if (phi_f2 < sp50[i].x) {
-                phi_f2 = sp50[i].x;
+            if (tempX > wallVertices[i].x) {
+                tempX = wallVertices[i].x;
+            } else if (temp < wallVertices[i].x) {
+                temp = wallVertices[i].x;
             }
 
-            if (sp44 > sp50[i].z) {
-                sp44 = sp50[i].z;
-            } else if (phi_f12 < sp50[i].z) {
-                phi_f12 = sp50[i].z;
+            if (tempZ > wallVertices[i].z) {
+                tempZ = wallVertices[i].z;
+            } else if (maxWallZ < wallVertices[i].z) {
+                maxWallZ = wallVertices[i].z;
             }
         }
 
-        sp4C = (sp4C + phi_f2) * 0.5f;
-        sp44 = (sp44 + phi_f12) * 0.5f;
+        // average min and max x/z of wall vertices
+        tempX = (tempX + temp) * 0.5f;
+        tempZ = (tempZ + maxWallZ) * 0.5f;
```

Comment:
Interesting that you decided to document `func_8083F0C8` as part of this PR, this function deals with entering crawlspaces and seems quite separate to equips. I also attempted to document this function but haven't submitted a PR with it or anything, feel free to compare :)
https://github.com/engineer124/oot/blob/1eb3c31f3807e98b728ac0af8dc54722ccd033b9/src/overlays/actors/ovl_player_actor/z_player.c#L6138

Reply:
I have no idea why this is in. Probably just random cleanup I included. player would be a lot more readable if we at least improved the variable names I guess

we have almost the same thing doc-wise I guess, I didn't know this is crawlspace related though. Your temp names for the floats may be better than mines, idk it's hard to gauge with how reused they all are.

I can revert this if you want, though I'm sure you could handle the merge conflict